src/main/java/com/bhavit/csvreadwrite/util/CsvHandler.java

```diff
@@ -12,10 +12,10 @@
 import java.io.IOException;
 
 /**
+ *
  * class that reads records from CSV file. and save to another CSV file.
```

Comment:
I apologize for the misunderstanding, but your request cannot be completed as you have not provided the Java code for review. Please provide the code, and I'll be more than happy to assist you!